README.md

```diff
@@ -4,79 +4,202 @@
-        :tags: that's, awesome
-        :category: yeah
-        :slug: my-super-post
-        :authors: Alexis Metaireau, Conan Doyle
-        :summary: Short version for index and feeds
-        ```
-
-        Please read [Writing content](https://docs.getpelican.com/en/latest/content.html) section in pelican documentation for format detail.
-2. After you finish a new article, you can run `pipenv run inv livereload`. It will build our current content into web pages, serve it on `http://localhost:8000/`, and open your default browser.
-3. If everything looks good, use the following command to add the new article to this repository.
-
-    ```sh
-    git add <your new article>
-
-    # use commitizen to do git commit
-    # choose the "new post" type if you're adding a new article
+#### Quick Start
+
+1. Create Post
+
+    Run the following command to create a new post. Follow the steps to fill in all necessary information.
+
+    ```bash
+    python create_post.py
+    ```
+
+    Then open the newly created file under `content/posts` to finish editing the rest of the content body.
+    The post is written in Markdown format. You can learn more about Markdown [here](https://www.markdownguide.org/cheat-sheet/).
+
+2. Test Locally
+
+    It's very ***IMPORTANT*** to test and run locally before committing anything. Run the following command to host the website locally.
+
+    ```bash
+    pipenv run inv livereload
+    ```
+
+    After executing the above command, open your browser and navigate to `http://localhost:8000/`.
+    You should be able to see the new post you've just created.
+
+3. Commit
+
+    After all is ready, it's time to commit the modifications to the branch and push to the repository.
+
+    ```bash
+    git add <your_file>
+
+    # Use commitizen to do git commit.
+    # Choose the "new post" type if you're adding a new article
     pipenv run cz commit
 
-    git push origin <your feature branch>
+    # Push to the remote branch.
+    git push origin $(git rev-parse --abbrev-ref HEAD)
     ```
-4. Create a pull request to [pycontw/pycontw-blog](https://github.com/pycontw/pycontw-blog).
+
+4. Create Pull Request
+
+    After pushing to the remote, go back to your github page of the forked repository. There should be a very obvious pop up on top of the page like below
+    <details>
+    <summary>Click me</summary>
+    <img src="content/images/pr_step_1.png" />
+    </details>
+
+    Press that `Compare & pull request` hardly and go to the next page.
+
+    <details>
+    <summary>Click me</summary>
+    <img src="content/images/pr_step_2.png" />
+    </details>
+
+    Modify the content of the red rectangle 1 and 2. After editing, press the `Create pull request` button. That's it!!
+
+Congratulations!! You've done all the jobs to post a new blog article. The next is taking a rest, drinking a cup of tea, and waiting for the maintainer to come for reviewing your PR ~
+
+
+#### More about the post
```

Comment:
This section looks great! It might worth mentioning the pelican doc so the contributor will know where to find the details